boards/nrf52840-dongle/Makefile.include

```diff
@@ -0,0 +1,39 @@
+# This board uses the vendor's serial bootloader
+
+PROGRAMMER ?= nrfutil
+
+# We're not using the softdevice actually (nor use JLink as in
+# nrf52/Makefile.include), but the linker script should be a good starting
+# point for using the nRF bootloader and not breaking the soft device.
+
+LINKER_SCRIPT ?= $(BOARDSDIR)/nrf52840-dongle/ldscripts/application.ld
```

Comment:
I don't think the custom linker script is neccessary. Have you tried simply setting `ROM_OFFSET ?= 0x1000` (see e.g. `/boards/sensebox_samd21/Makefile.include` -> this board is using the Arduino bootloader...). This should actually force the generic linkerscript in RIOT to believe that the flash starts at addr 0x1000, leaving the nordic bootloader intact.

Reply:
Aaeh ... yes, I take it; tried now and it works fine. Updated, thanks for the pointer.